strongbox-data-service/src/main/java/org/carlspring/strongbox/data/criteria/Predicate.java

```diff
@@ -23,6 +23,10 @@
 
     private List<Predicate> childPredicateList = new ArrayList<>();
 
+    private boolean nesteed;
```

Comment:
There's a small typo here. You mean `nested`. :)
Could you please fix the getter/setter as well? Thanks! :)

Reply:
fixed